src/dbnode/persist/fs/seek_manager.go

```diff
@@ -123,6 +123,26 @@ type seekersAndBloom struct {
 	volume      int
 }
 
+func (s seekersAndBloom) close() error {
+	multiErr := xerrors.NewMultiError()
+	for _, seeker := range s.seekers {
+		multiErr = multiErr.Add(seeker.seeker.Close())
+	}
+	return multiErr.FinalError()
+}
+
+// Returns true if any seekers are borrowed.
+func (s seekersAndBloom) anyBorrowed() bool {
+	anySeekersAreBorrowed := false
+	for _, seeker := range s.seekers {
+		if seeker.isBorrowed {
+			anySeekersAreBorrowed = true
```

Comment:
nit: Perhaps just return true here and then return false at end if early return true?